spec/cobrato/entities/bank_billet_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+describe Cobrato::Entities::BankBillet do
+  let(:attributes) do
+    {
+      bank_code: '001',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.